core/ui/EditorToolbar/picture-dropdown.tid

```diff
@@ -4,8 +4,44 @@ title: $:/core/ui/EditorToolbar/picture-dropdown
 [img[$(imageTitle)$]]
 \end
 
+\define insert-external-image()
```

Comment:
Oh man that looks awesome, seems to be doing everything I envisioned.

I'll give it a more thorough look later tonight when I'm back at my desktop.
Thanks for all the hard work :)